trie/verkle.go

```diff
@@ -220,11 +219,11 @@ func (trie *VerkleTrie) TryDelete(key []byte) error {
 // Hash returns the root hash of the trie. It does not write to the database and
 // can be used even if the trie doesn't have one.
 func (trie *VerkleTrie) Hash() common.Hash {
-	return trie.root.ComputeCommitment().Bytes()
+	return trie.root.Commit().Bytes()
 }
 
 func nodeToDBKey(n verkle.VerkleNode) []byte {
-	ret := n.ComputeCommitment().Bytes()
+	ret := n.Commit().Bytes()
```

Comment:
as stated before, the difference between `Commitment` and `Commit` will disappear when my refactor lands in `master`, but I would like this one to be `Commitment` as it would help me capture an error if the value hasn't been computed. The one just above should be `Commit`